src/server/iframe.html.js

```diff
@@ -3,6 +3,11 @@ export default function (headHtml) {
     <!DOCTYPE html>
     <html>
       <head>
+        <script>
+          if (window.parent !== window) {
```

Comment:
It should really be default behaviour: As React developers we expect the dev tools to "just work". 
@wyattdanger Thank for doing the PR; saved me a job.